watch.go

```diff
@@ -21,7 +23,23 @@ type Event struct {
 	// Path to the data that changed
 	Path string
 	// Data that changed
-	Data interface{}
+	Data    interface{}
+	payload string
+}
+
+func (e Event) Value(v interface{}) (path string, err error) {
```

Comment:
Sure. I will do it as soon as the question concerning the path as a return
value is closed.

█  Jacques

On 2 August 2016 at 04:29, Steven Berlanga <notifications@github.com> wrote:

> In watch.go
> <https://github.com/zabawaba99/firego/pull/50#discussion_r73085481>:
>
> > @@ -21,7 +23,23 @@ type Event struct {
> >  	// Path to the data that changed
> >  	Path string
> >  	// Data that changed
> > -	Data interface{}
> > +	Data    interface{}
> > +	payload string
> > +}
> > +
> > +func (e Event) Value(v interface{}) (path string, err error) {
>
> Also, could you add documentation for this function?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zabawaba99/firego/pull/50/files/91637d68c23bcd6761930b23882b18621b759365#r73085481>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAB2iJW2_yBnFy_3qB_6NX5vnUo0-kpiks5qbquGgaJpZM4JYHTS>
> .
>